### ISSUE TITLE
Got an uninitialized constant InfluxDB2::Query::Set error

### ISSUE THREAD

**Issue by karn18** [influxdata/influxdb-client-ruby#84]
__Steps to reproduce:__
I've tried to query the data with synchronous query example
```
client = InfluxDB2::Client.new('https://localhost:8086', 'my-token',
                              bucket: 'my-bucket',
                              org: 'my-org')

query_api = client.create_query_api
result = query_api.query(query: 'from(bucket:"' + bucket + '") |> range(start: 1970-01-01T00:00:00.000000001Z) |> last()')
```

__Expected behavior:__
I expected to see the result.

__Actual behavior:__
Got an error:

```
/Users/karn/.rbenv/versions/3.0.1/lib/ruby/gems/3.0.0/gems/influxdb-client-1.15.0/lib/influxdb2/client/models/query.rb:77:in `openapi_nullable': uninitialized constant InfluxDB2::Query::Set (NameError)
Did you mean?  Net
        from /Users/karn/.rbenv/versions/3.0.1/lib/ruby/gems/3.0.0/gems/influxdb-client-1.15.0/lib/influxdb2/client/models/query.rb:260:in `block in to_hash'
        from /Users/karn/.rbenv/versions/3.0.1/lib/ruby/gems/3.0.0/gems/influxdb-client-1.15.0/lib/influxdb2/client/models/query.rb:257:in `each_pair'
        from /Users/karn/.rbenv/versions/3.0.1/lib/ruby/gems/3.0.0/gems/influxdb-client-1.15.0/lib/influxdb2/client/models/query.rb:257:in `to_hash'
        from /Users/karn/.rbenv/versions/3.0.1/lib/ruby/gems/3.0.0/gems/influxdb-client-1.15.0/lib/influxdb2/client/models/query.rb:250:in `to_body'
        from /Users/karn/.rbenv/versions/3.0.1/lib/ruby/gems/3.0.0/gems/influxdb-client-1.15.0/lib/influxdb2/client/query_api.rb:76:in `_post_query'
        from /Users/karn/.rbenv/versions/3.0.1/lib/ruby/gems/3.0.0/gems/influxdb-client-1.15.0/lib/influxdb2/client/query_api.rb:59:in `query_stream'
        from client.rb:22:in `<main>'
```

__Specifications:__
 - Client Version: 1.15.0
 - InfluxDB Version: 2.0.7
 - Platform: docker

**Comment by bednar** [comment]
@karn18, thanks for using our client.

It looks like problem related to Ruby `v3.0.1`. I've tested it for ruby `ruby 2.6.3p62` and everything works fine.

Regards

**Comment by bednar** [comment]
The #86 fixes compatibility for Ruby `v3.0.1`.
